cmd/kubeadm/app/cmd/options/generic.go

```diff
@@ -80,3 +82,10 @@ func AddKubernetesVersionFlag(fs *pflag.FlagSet, kubernetesVersion *string) {
 		`Choose a specific Kubernetes version for the control plane.`,
 	)
 }
+
+// AddOutputFlag adds the --output/-o flag to the given flagset
+func AddOutputFlag(fs *pflag.FlagSet, outputFormat *string) {
+	fs.StringVarP(
+		outputFormat, "output", "o", output.TEXTOutput,
```

Comment:
Might start with a `--experimental-output`. Also, not sure if `o` would be available for an alpha feature. Might want to just reserve it for now (but this needs discussion).

Reply:
I thought that it's safe to add new option as we're not modifying existing option here. It's not a big deal to change though. Agree, we need to discuss this.

Reply:
+1 for `--experimental-output` and `-o`

Reply:
done